index.js

```diff
@@ -90,7 +90,7 @@ module.exports = (apiUrl, apiKey, options = {}) => {
 
         log(`Response from GET ${url}`)
         yield response.body
-        url = response.headers && getNextUrl(response.headers.link)
```

Comment:
This is a fix when `headers` are present but `headers.link` not. Not strictly related to the `better-testing` branch